units/host_allocator.go

```diff
@@ -307,3 +286,33 @@ func (j *hostAllocatorJob) Run(ctx context.Context) {
 		"runner":                       scheduler.RunnerName,
 	})
 }
+
+func (j *hostAllocatorJob) setTargetAndTerminate(ctx context.Context, lenUpHosts int, hostQueueRatio float32, distro *distro.Distro) {
+	var killableHosts, newCapTarget int
+	if hostQueueRatio == 0 {
+		killableHosts = lenUpHosts
+		newCapTarget = 0
```

Comment:
nit: you don't need to explicitly set this to 0 since the default int value is 0.